src/content/docs/en/guides/deploy/clever-cloud.mdx

```diff
@@ -0,0 +1,125 @@
+---
+title: Deploy your Astro Site to Clever Cloud
+description: How to deploy your Astro site to the web on Clever Cloud.
+type: deploy
+i18nReady: true
+---
+import { Tabs, TabItem, Steps } from '@astrojs/starlight/components';
+
+[Clever Cloud](https://clever-cloud.com) is a European cloud platform that provides automated, scalable services. You can deploy your Astro app with Clever Cloud as a static or Node application.
+
+## Project Configuration
+
+Your Astro project can be deployed on Clever Cloud in two different ways: as a **Static** application, or as a **Node.js** application, regardless of your `output` mode ([pre-rendered or on-demand](/en/basics/rendering-modes/)). However, you'll need less configuration when deploying a Node.js application, since it works out-of-the-box with your `package.json`.
+
+### Port and host
+
+Applications on Clever Cloud listen on port **8080**. If your project requires this configuration, you can set your port and host in Astro in two ways:
+
+1. In `package.json` scripts:
+
+    ```json title="package.json" "astro preview --host 0.0.0.0 --port 8080"
+    "scripts": {
+      "dev": "astro dev",
+      "start": "astro dev",
+      "build": "astro check && astro build",
+      "preview": "astro preview --host 0.0.0.0 --port 8080",
+      "astro": "astro"
+    } 
+    ```
+
+2. In `astro.config.mjs`:
+
+    ```javascript title="astro.config.mjs"
+    import { defineConfig } from 'astro/config';
+
+    export default defineConfig({
+      server: {
+        port: 8080,
+        host: true
+      }
+    });
+    ```
+
+
+## Deploy Astro from the Console
+
+When deploying on Clever Cloud, follow the application wizard to deploy your Astro site. These are the detailed steps:
+
+<Steps>
+
+1. From the lateral menubar, click **Create** > **An application**
+2. Choose how to deploy:
+
+    - **Create a brand new app**: to deploy from a local repository with Git
+    
+    or
+    
+    - **Select a GitHub repository**: to deploy from GitHub
+
+3. Select a **Node.js** application, or a **Static** one.
+4. Set up the minimal size for your instance and scalability options: Astro can be deployed in a **Nano** instance as minimal size. Instance sizes depend on your application specifications and dependencies. We suggest you start with the smallest instance and then adjust accordingly as you watch the metrics from the **Overview** page.
+5. Select a **region** where to deploy your instance.
+6. **Add-ons**: Unless you're using a database or Keycloak, you shouldn't need any add-ons. See [Further Configuration](#further-configuration) if you do.
```

Comment:
```suggestion
6. **Add-ons**: Unless you're using a database or Keycloak, you shouldn't need any add-ons.
```
The Further Configuration section doesn't appear to exist, so maybe it was removed in an earlier edit. Just fixing the link error!